app/api-v1/api-doc.js

```diff
@@ -0,0 +1,59 @@
+const { PORT, API_VERSION, API_MAJOR_VERSION } = require('../env')
+
+const apiDoc = {
+  openapi: '3.0.3',
+  info: {
+    title: 'IdentityService',
+    version: API_VERSION,
+  },
+  servers: [
+    {
+      url: `http://localhost:${PORT}/${API_MAJOR_VERSION}`,
```

Comment:
This probably shouldn't be localhost. Are we planning on using the openapi merge methodology we used in WASP to override this?

Reply:
Still TBD

Reply:
Yeah that's fine. For now we can leave it and make this decision down the line